homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -216,7 +279,7 @@ async def async_set_volume(self, volume):
         await self._handle_errors(self._control.set_volume, volume)
 
     async def async_play_media(self, media_type, media_id):
-        """Play media."""
+        """Open webpage."""
```

Comment:
Why change the docstring to this?

Reply:
It was because `async_play_media` can only open a webpage

I'll revert it back!